client/src/components/Home/Home.jsx

```diff
@@ -4,20 +4,21 @@ import {Link} from "react-router-dom"
 import axios from "axios"
 import { useState, useEffect } from "react"
 
-export default function Home({sessionToken}) {
+export default function Home({sessionToken, recs}) {
     const [recents, setRecents] = useState([])
 
+
     async function getRecent() {
         try {
             const recentResponse = await axios.get(`http://localhost:3001/books/recent/${sessionToken}`)
             setRecents(recentResponse.data)
         }
         catch (err) {
-            alert("Couldn't load recommendations.")
+            alert("Couldn't load recent activity.")
         }
     }
     useEffect(() => {
-        getRecent()
+        {if (sessionToken!==null) getRecent()}
```

Comment:
In which case would sessionToken be null?

Reply:
sessionToken being null would mean no one has logged in, so I wouldn't want to get the user's recent activity in that case.